### PR TITLE
restore: document third-party repos

### PR DESCRIPTION
**Pull request by gsanchietti**
See https://community.nethserver.org/t/external-repository-not-restored-from-configuration-restore/16558

### CODE REVIEW

File: administrator-manual/en/disaster_recovery.rst [hunk 1]
```diff
@@ -19,6 +19,14 @@ Other restored configurations:
    installation, i.e., avoid restoring a configuration backup from a 7.4.1708 installation on a new 7.6.1810 system,
    as it may lead to unexpected results.
 
+.. only:: nscom
+
+   .. note::
+
+       Third-party repositories will not be restored by disaster recovery procedure.
+       If the original machine had some third-party repository enabled, remember to install them before
+       proceeding with the restore.
```
Comment (DavidePrincipi) [inline administrator-manual/en/disaster_recovery.rst:28]:
```suggestion
       Third-party repositories are not restored by the disaster recovery procedure.
       If the original machine has some third-party repository enabled, remember to install it before
       proceeding with the restore.
```

**Review by DavidePrincipi — COMMENTED** [review]
I'm not sure about the right verb tenses, but simple present should work...

File: administrator-manual/en/disaster_recovery.rst (hunk 1 above)
Comment (gsanchietti) [reply administrator-manual/en/disaster_recovery.rst:28]: I agree, it's much better!

**Review by DavidePrincipi — APPROVED** [review]

**Comment by DavidePrincipi** [comment]
I still have some doubts about the "if" sentence :thinking: 

Hopefully someone else will fix it...

**Comment by gsanchietti** [comment]
Let's aks @filippocarletti  ;)

**Pull request merged**